software-update/update.html

```diff
@@ -0,0 +1,25 @@
+<head>
+<title>Software-update</title>
+ <meta charset="utf-8">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+    <script type="text/javascript">
+
+function fun1()
+{
+            var proc = cockpit.spawn("aero-reboot-update.py");
+            proc.fail(On_fail);
+        }
```

Comment:
this function is failing parser... ?  Also check your spaces... indent should be 4 spaces.